components/Buttons/src/MDCFloatingButton.m

```diff
@@ -19,9 +19,49 @@
 #import "MaterialShadowElevations.h"
 #import "private/MDCButton+Subclassing.h"
 
+#import <MDFInternationalization/MDFInternationalization.h>
+
 static const CGFloat MDCFloatingButtonDefaultDimension = 56.0f;
 static const CGFloat MDCFloatingButtonMiniDimension = 40.0f;
+static const CGFloat MDCFloatingButtonDefaultImageTitleSpacing = 8;
+static const UIEdgeInsets internalLayoutSpacingInsets = (UIEdgeInsets){0, 16, 0, 24};
+
 static NSString *const MDCFloatingButtonShapeKey = @"MDCFloatingButtonShapeKey";
+static NSString *const MDCFloatingButtonModeKey = @"MDCFloatingButtonModeKey";
+static NSString *const MDCFloatingButtonImageLocationKey = @"MDCFloatingButtonImageLocationKey";
+static NSString *const MDCFloatingButtonImageTitleSpacingKey = @"MDCFloatingButtonImageTitleSpacingKey";
+
+static UIEdgeInsets UIEdgeInsetsFlippedHorizonally(UIEdgeInsets insets) {
```

Comment:
FYI: This also exists as MDFInsetsFlippedHorizontally() in MDFI18n.  Also this will cause an error if our friends in Cupertino ever decide to declare UIEdgeInsetsFlippedHorizonally.  Although they may throw in a 't' 😄

Reply:
Done